README.md

```diff
@@ -22,24 +22,20 @@ Example usage:
     logbuddy https://example.com/logs.txt --model=https://mymodel.co/model_file --summarizer=drain --nlines=10
 
 
-Development
------------
+Contributing
+------------
 
-To develop the script, clone this repository and install dependencies using pip:
+Contributions are welcome! Please submit a pull request if you have any improvements or new features to add. Make sure your changes pass all existing tests before submitting.
 
-git clone https://github.com/yourusername/logbuddy.git
-cd logbuddy
-pip install -r requirements.txt .
+To develop the script, clone this repository and install dependencies using pip:
```

Comment:
```suggestion
To develop logbuddy, you should fork this repository, clone your fork, and install dependencies using pip:
```
I know it's nitpicking, but just to be precise.

Reply:
Done